api/src/test/java/org/openmrs/util/databasechange/Database2_2_0UpgradeIT.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util.databasechange;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.openmrs.util.DatabaseUtil;
+import org.openmrs.util.OpenmrsConstants;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.StringReader;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests database upgrade from OpenMRS 2.2.0.
+ */
+public class Database2_2_0UpgradeIT extends BaseContextSensitiveTest {
+	
+	public static final String TEST_DATA_DIR = "/org/openmrs/util/databasechange/";
+	
+	public static final String UPGRADE_TEST_2_1_0_TO_2_20_DATASET = TEST_DATA_DIR
+		+ "database2_1To2_20UpgradeTest-dataSet.xml";
+	
+	public static final String STANDARD_TEST_2_2_0_DATASET = TEST_DATA_DIR + "standardTest-2.2.0-dataSet.xml";
+	
+	public final static String DATABASE_PATH = TEST_DATA_DIR + "openmrs-2.2.0.h2.db";
```

Comment:
@wluyima 
How do I create such a .db file in the first place? I have no idea how this is done? So far I just duplicated the 1.9.0 integration test. I do no know what steps are necessary to write such a test class. I am just trying to port the 1.9.0 test as good as I can since that was required after the last PR version.

Reply:
You are writing a test to demonstrate that one can upgrade from 2.1.x to 2.2.0, therefore your base dataset should be a 2.1.x dataset, the h2 documentation must have steps on how to create the .db file.

Reply:
I am still not sure how such a file can be created out of the Hibernate definitions and for what version this should appear. I posed regarding this concern in Talk:

https://talk.openmrs.org/t/ra-865-enable-running-openmrs-2-x-on-h2-database/2616/3